src/rust/sdk/src/tx_builder.rs

```diff
@@ -89,6 +115,13 @@ impl<'a> EconiaTransactionBuilder<'a> {
         self
     }
 
+    /// Set the internal [EntryFunction] as [upgrade_integrator_fee_store_via_coinstore](https://github.com/econia-labs/econia/blob/dev/src/move/econia/doc/incentives.md#0xc0deb00c_incentives_upgrade_integrator_fee_store_via_coinstore)
```

Comment:
Can we parameterize the `c0deb00c` section of the URL? That is subject to change with `optivanity`

Reply:
I don't think this is possible? @bogberry do you think it is?

Reply:
@qdrs @bogberry I just realized if we need to change things in the future we can do find and replace all, so this shouldn't be an issue.